src/react-apps/applications/service-development/src/features/appPublish/containers/deployContainer.tsx

```diff
@@ -11,12 +13,27 @@ import ConfigurationActions from '../../../sharedResources/configuration/configu
 import { IConfigurationState } from '../../../sharedResources/configuration/configurationReducer';
 import AppDeploymentComponent from '../components/appDeploymentComponent';
 
-export interface IDeployContainer {
+const theme = createMuiTheme(StudioTheme);
+
+const styles = createStyles({
+  deployContainer: {
+    overflow: 'scroll',
+    [theme.breakpoints.up('xs')]: {
+      height: 'calc(100vh - 5.5rem)',
+    },
+    [theme.breakpoints.up('md')]: {
+      height: 'calc(100vh - 11.1rem)',
```

Comment:
If the AppBar is 110px... should this also pe 110px?

Reply:
The AppBar is 110px + 1px border on the bottom.